### PR TITLE
Schema Update

### PR DESCRIPTION
**Pull request by DeeptanshuPrasad**
> Fixed image not showing up in schema for Blog Posts.
> Fixed organisation schema for individual pages.
> Validated schema with validator.schema.org

### CODE REVIEW

File: pages/blog/_slug.vue [hunk 1]
```diff
@@ -292,14 +292,11 @@ export default {
       },
       headline: this.article.title,
       description: this.article.description,
-      metaTitle: this.article.metaTitle,
-      metaDescription: this.article.metaDescription,
```
Comment (ravin1227) [inline pages/blog/_slug.vue:296]: Don't we need `metaTitle` and `metaDescription` for schema ?

File: pages/features/auto-responder.vue [hunk 2]
```diff
@@ -204,25 +248,10 @@ export default {
...
-      creator: {
-        '@type': 'Organization',
-        '@id': '#organization',
-        url: 'https://formester.com/',
-        name: 'Formester',
-        logo: {
-          '@type': 'ImageObject',
-          url: 'https://formester.com/logo.png',
-        },
-      }
```
Comment (ravin1227) [inline pages/features/auto-responder.vue:225]: Are these not required anymore?

File: pages/blog/_slug.vue (hunk 1 above)
Comment (DeeptanshuPrasad) [reply pages/blog/_slug.vue:296]: Yes, we validated this through validator.schema.org, metaTitle and metaDescription are not required in schema. 
Screenshot 2023-05-25 at 12 41 18 PM

File: pages/features/auto-responder.vue (hunk 2 above)
Comment (DeeptanshuPrasad) [reply pages/features/auto-responder.vue:225]: There were duplicated entries for address and creator in some places, hence removed them. Also, 'creator' is not required in organisation schema, hence removed that too.

Comment (ravin1227) [reply pages/features/auto-responder.vue:225]: okay

File: pages/blog/_slug.vue (hunk 1 above)
Comment (ravin1227) [reply pages/blog/_slug.vue:296]: okay

**Review by ravin1227 — APPROVED** [review]

**Pull request merged**